lib/mongo/monitoring.rb

```diff
@@ -328,14 +328,20 @@ def publish_heartbeat(server, awaited: false)
       rescue => exc
         if monitoring?
           event = Event::ServerHeartbeatFailed.new(
-            server.address, Time.now-start_time, exc, awaited: awaited)
+            server.address, Time.now-start_time, exc,
```

Comment:
Bit of a nit but I would expect all parameters do be on new lines or not - not a combination of them.